ftp/src/main/scala/akka/stream/alpakka/ftp/FtpsSource.scala

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.stream.alpakka.ftp
```

Comment:
I think we have refactored the package conventions after you started this. We now use `scaladsl` and `javadsl`. See the other connectors in master.